webapp/javascript/components/ExportData.tsx

```diff
@@ -44,6 +49,21 @@ function ExportData() {
     setToggleMenu(!toggleMenu);
   };
 
+  const downloadFlamebearer = function (
+    exportObj: Flamebearer,
+    exportName: string
+  ) {
+    const dataStr = `data:text/json;charset=utf-8,${encodeURIComponent(
+      JSON.stringify(exportObj)
```

Comment:
for large files, could we do use something like this? https://github.com/eligrey/FileSaver.js

Reply:
Good idea, I'm going to be bold here and leave as it is until it become a problem though. I believe our flamegraphs are small enough for that to become an issue.